.github/workflows/wheels-dependencies.sh

```diff
@@ -150,12 +137,8 @@ function build_libavif {
             -DBUILD_SHARED_LIBS=OFF \
             -DAVIF_LIBSHARPYUV=LOCAL \
             -DAVIF_LIBYUV=LOCAL \
-            -DAVIF_CODEC_RAV1E=SYSTEM \
-            -DAVIF_CODEC_AOM=LOCAL \
-            -DAVIF_CODEC_DAV1D=LOCAL \
-            -DAVIF_CODEC_SVT=LOCAL \
```

Comment:
This sort of change I think I would rather have in a separate pull request. You are aware that rav1e is only an encoder, and so if we don't include either aom or dav1d, we can't decode AVIF images?

Reply:
Ah, ok, I missed that libavif was installing the dependencies itself when using "LOCAL".

I've created https://github.com/fdintino/Pillow/pull/7 to use "LOCAL" for rav1e, except on manylinux2014 and aarch64.